### ISSUE TITLE
No response from server

### ISSUE THREAD

**Issue by D3laru3** [vrpn/vrpn#135]
Hi.
I am using the last version from the repository but when I want to print_devices Xbox or just keyboard0 devices it shows me a warning "No response from the server for >= 3 seconds". With mouse0, it works.

Could you please help?

Thanks

**Comment by russell-taylor** [comment]
I'm afraid I don't have enough information to be certain that I can help.  I infer from the statement "With mouse0, it works" that you are running vrpn_server with a configuration file on either Windows or Linux.

Potential reasons why mouse work work and keyboard not: (1) You could be running on non-Windows platforms, which should print an error when you run the server on another platform.  (2) You could be using a different name for the device in the VRPN config file (Keyboard0 by default) and the vrpn_print_devices argument.

The fact that the client cannot connect to the server indicates that you maybe the right name (Keyboard0@localhost for keyboard, microsoft0@localhost for the Xbox) is needed, or that the appropriate line in vrpn.cfg is not uncommented (run vrpn_server -v to get verbose print out of what devices are opened), or that you need to bypass a firewall using microsoft0@tcp:localhost.

**Comment by D3laru3** [comment]
thanks for your reply. I already tried to run vrpn_server -v and I only get Mouse0 even if i commented it...I am not using different names I am using default values (Mouse0, XInput0 etc) and uncommented it...I am using windows 10...its confusing that mouse work and other devices dont. I am using last release from git and built server on visual studio...after that i built print_devices and copy the vrpn.cfg to vrpn_server.exe folder. After that I uncommented mouse, keyboard, and XInput...vrpn_server still showing only Mouse0
